### PR TITLE
[IGA] Scordelis Lo roof tests for shell_3p elements

### PR DESCRIPTION
**Pull request by rickyaristio**
**Description**
In this PR, I have added the Scordelis roof tests for the KL-3p shell. The first test is cpp unit tests, and the second one is a complete test using the Iga test factory.

One remark, this branch is merged with #7087, as it is needed for the tests. 

**Changelog**
- test_iga_shell_3p_Scordelis_roof.cpp
- scordelis_roof_shell_3p_test

### CODE REVIEW

File: applications/IgaApplication/tests/cpp_tests/test_creation_utility_Scordelis_roof.h [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License: BSD License
+//           Kratos default license: kratos/license.txt
+//
+
+#if !defined(KRATOS_IGA_TEST_CREATION_UTILITY )
```
Comment (tteschemacher) [inline applications/IgaApplication/tests/cpp_tests/test_creation_utility_Scordelis_roof.h]: you need to define a unique define

File: applications/IgaApplication/tests/test_modelers.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+from __future__ import print_function, absolute_import, division
```
Comment (dagmawibekel) [inline applications/IgaApplication/tests/test_modelers.py]: This is not needed anymore, because python2 is not supported

Comment (rickyaristio) [reply applications/IgaApplication/tests/test_modelers.py]: Okay

**Comment by tteschemacher** [comment]
Hey @dagmawibekel could you have a look here. It is showing a way how we will create cpp tests in the future for our IGA elements.

**Comment by tteschemacher** [comment]
ping @dagmawibekel

**Review by dagmawibekel — APPROVED** [review]

**Comment by tteschemacher** [comment]
sweet thanks. And a big thank you to @rickyaristio for preparing this PR.

**Pull request merged**